src/setupTests.js

```diff
@@ -1,5 +1,13 @@
+/* eslint-disable */
 // jest-dom adds custom jest matchers for asserting on DOM nodes.
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom/extend-expect'
+
+const localStorageMock = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  clear: jest.fn()
+}
+global.localStorage = localStorageMock
```

Comment:
This is the jest-approved way of mocking localStorage globally. Unfortunately this alone doesn't solve the redux persist issue.